firstvoices/backend/tests/test_apis/test_pages_api.py

```diff
@@ -90,7 +90,7 @@ def get_expected_response(self, instance, site):
         ] = f"http://testserver{self.get_detail_endpoint(key=instance.slug, site_slug=site.slug)}"
         return {
             **controlled_standard_fields,
-            "subtitle": "",
+            "subtitle": None,
```

Comment:
Instead of changing this, please add a new test to check support for the null value